mypy_tests.ini

```diff
@@ -13,6 +13,19 @@ ignore_errors = True
 [mypy-pydantic.*]
 ignore_errors = True
 
+[mypy-sqlalchemy.*]
+ignore_errors = True
+check_untyped_defs = False
+
+[mypy-sqlmodel.*]
+ignore_errors = True
+
+[mypy-ormar.*]
+ignore_errors = True
+
+[mypy-databases.*]
+ignore_errors = True
+
```

Comment:
Should probably have a discussion on the implications of this. What instigated these ignores?

Reply:
As I understand, they silent typing errors from dependencies I added (ormar and slqmodel), and I don't know if we can do much about it.